modules/yelp.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const superagent = require('superagent');
+const client = require('../utilities/db');
+const errorHandler = require('./error');
+
+function yelpHandler(request, response) {
+  console.log(request.query);
+  const lat = request.query.latitude;
+  const lon = request.query.longitude;
+  // const restaurants = request.query.categories=restaurants;
+  const yelpURL = 'https://api.yelp.com/v3/businesses/search';
+  superagent.get(yelpURL)
+    .set('Authorization', 'Bearer ' + process.env.YELP_KEY)

```

Comment:
Use a template literal?
```suggestion
    .set('Authorization', `Bearer ${process.env.YELP_KEY}`)
```